cs/routing.texy

```diff
@@ -407,6 +407,16 @@ $route = new Route('<presenter>/<action>', [
 *Globální filtry* můžeme použít též pro filtrování parameterů na základě jiných parametrů. Například přeložení `<presenter>` a `<action>` na základě parametru `<lang>`.
 
 
+Cachování rout
+--------------
+
+Definované routy lze nechat automaticky serializovat do vygenerovaného DI kontejneru, pouhým povolením v konfiguračním souboru.
```

Comment:
Myslím, že nikdo nepochopí o co go.

Jde o to, že router se sestaví jen jednou a poté se do kontejneru dá jeho serializovaná verze. Takže při každém požadavku je sestavení maximálně rychlé, ale omezení je, že router nesmí mít žádné závislosti, jako třeba na databázi apod.

Reply:
Zkusil sem to preformulovat.